pkg/capabilities/consensus/ocr3/types/ocr3_types.proto

```diff
@@ -36,6 +37,8 @@ message Observation {
 message Observations {
   // batched observations for multiple workflow execution IDs
   repeated Observation observations = 1;
+  // the workflow IDs that are registered in the node
+  repeated string registeredWorkflowIds = 2;
```

Comment:
why does this need to be in the proto now? specifically, why is a field in Observations? it seems out of place. i would have expected either an api to list all the workflows or the observation itself has a workflow id

Reply:
See my answer below: this is so that we get consensus from all nodes on which workflows they think are registered to the capability